app/src/main/java/org/fossasia/pslab/adapters/PerformExperimentAdapter.java

```diff
@@ -112,6 +115,9 @@ public Fragment getItem(int position) {
                 if (experimentTitle.equals(context.getString(R.string.nfet_output_characteristics)))
                     return NFETOutputCharacteristicsExperiment.newInstance();
                 if (experimentTitle.equals("Half Wave Rectifier"))
+                    return RectifierExperiment.newInstance();
```

Comment:
Ah thanks for the review @viveksb007 It must have happened when resolving conflicts! I made the changes!